### DISCUSSION TITLE
Classroom autograding reporting error

### DISCUSSION THREAD

**Discussion by env3d** [community/community#138312] [GitHub Education]
### Hi everyone,

Hello classroom community, I'm having a strange issue with grade reporting from the csv download (and also api access).

I have setup autograding and the numerical grade is displayed as a double (see screenshot 1).  However, when I tried to download the csv file, all grades are report as 0.  See screenshot 2.

In a different classroom where the grades are integers, everything is working as expected.

Anyone having this issue?
<img width="548" alt="screenshot1" src="https://github.com/user-attachments/assets/4a332741-b73c-4567-bbbd-2bcaf08de93a">
<img width="1023" alt="screenshot2" src="https://github.com/user-attachments/assets/e6cd8c9a-8948-4140-95ea-6b9fdc880fa0">

**Comment by nelsonlim** [comment]
I can confirm we are seeing this weird rounding issue on our side too.

**Comment by bellecp** [comment]
This is still happening today. It makes exportation of grades impossible.

**Reply by beltransen** [reply]
Today, I encountered this same issue. I have tried both downloading the .csv using the webpage button and through the GitHub Classroom CLI command _gh classroom assignment-grades_. Either way, the results are the same and the exported grades don't match the actual grades displayed for every student in the assignment roster.

**Comment by gemoran** [comment]
I am also having this issue (Feb 2025)
